contracts/SocialProofs.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.4.0;
+
+contract SocialProofs {
+  event ProofEvent(address indexed userAddress, bytes16 indexed platformName, bytes data);
```

Comment:
bytes16 is not cheaper, the smallest storage size is 32.

just use bytes32 for platformName